tests/gui/pages/main.spec.js

```diff
@@ -223,222 +222,6 @@ describe('Browser: MainPage', function() {
         m.chai.expect(controller.getProgressButtonLabel()).to.equal('Flash!');
       });
 
-      describe('given there is a flash in progress', function() {
```

Comment:
Can we keep just one test case that tests something other than "Flash!" to cover the other side of the conditional?